src/app/helptext/plugins/plugins.ts

```diff
@@ -14,6 +14,9 @@ href="https://kb.iu.edu/d/adov" \
 target="_blank">DHCP</a> to automatically configure \
 network settings.'),
 
+nat_placeholder: T('NAT'),
+nat_tooltip: T(''),
```

Comment:
Network Address Translation (NAT). Transforms local network IP addresses into a single IP address. Set when the jail will share a single connection to the Internet with other systems on the network.